[docs] Improve docs on how to contribute to the docs (#4000)
drud/ddev: docs/content/developers/testing-docs.md

# Working on the docs

This section is designed for people who want to contribute to the DDEV documentation.
When you are going to make any changes to the documentation it is recommended that you test them locally to see what your changes look like.

## Fork / clone the DDEV repository

> To start making changes you need to get a local copy of the DDEV documentation, we recommend you to [fork the DDEV repository](https://github.com/drud/ddev/fork) as that's where the documentation lives.

Reviewer: ```suggestion
To start making changes you'll need a local copy of the DDEV documentation, so [fork the DDEV repository](https://github.com/drud/ddev/fork) as that's where the documentation lives.
```
